app/fissile.go

```diff
@@ -842,3 +877,255 @@ roleLoop:
 
 	return nil
 }
+
+func (f *Fissile) validate(roleManifest *model.RoleManifest, opinions *model.Opinions) validation.ErrorList {
```

Comment:
The 2nd one may cause a circle in the package imports.
Splitting out under `app/` should be easier.